src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: Person | null,
+  findParent: (name: string | null) => Person | null,
+};
+
+export const PersonInfo:React.FC<Props> = ({ person, findParent }) => {
+  const { personId } = useParams();
+
+  const mother = findParent(person?.motherName ? person?.motherName : null);
+  const father = findParent(person?.fatherName ? person?.fatherName : null);
+
+  const isMom = person?.motherName || '-';
+  const isDad = person?.fatherName || '-';
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
```

Comment:
nb: better to call this import for classNames like `cn` or `cx` (most common usage in real projects)